webui/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
Just to double-check this was intended.

This gives complete access to robots, whereas `Disallow: /` will block all access.

Reply:
Thanks, added `/`